plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityAwake.java

```diff
@@ -0,0 +1,41 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.entity.Bat;
+
+public class EntityAwake extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name awake
+    // @input ElementTag(Boolean)
+    // @description
+    // If the entity is a bat, controls whether they are flying (awake) or hanging (asleep).
```

Comment:
Should prefer the `Controls whether a bat is X` format, because then the meta site auto-replaces it to `Returns` and `Sets` for the tag/mechanism docs - otherwise lgtm